### DISCUSSION TITLE
Update OpenCL in Jellyfin LXC (for HW Transcoding)

### DISCUSSION THREAD

**Discussion by dimitrij2k** [tteck/Proxmox#3112] [Ideas]
Hardware: Intel N100 Mainboard (Intel 12th Gen)

GPU Support out of the Box ✅
GPU Usage out of the Box ❌ (for example no transcoding possible, only black screen)

How do i know? `intel_gpu_top` everything stay at 0% and blackscreen when change Resolution of Video to 720p for example

Why? Missing newest OpenCL Driver!

How to install:
https://github.com/intel/compute-runtime/releases

Summarized:

1. `mkdir openclupdate`
2. `cd openclupdate
wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.16695.4/intel-igc-core_1.0.16695.4_amd64.deb
wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.16695.4/intel-igc-opencl_1.0.16695.4_amd64.deb
wget https://github.com/intel/compute-runtime/releases/download/24.17.29377.6/intel-level-zero-gpu-dbgsym_1.3.29377.6_amd64.ddeb
wget https://github.com/intel/compute-runtime/releases/download/24.17.29377.6/intel-level-zero-gpu_1.3.29377.6_amd64.deb
wget https://github.com/intel/compute-runtime/releases/download/24.17.29377.6/intel-opencl-icd-dbgsym_24.17.29377.6_amd64.ddeb
wget https://github.com/intel/compute-runtime/releases/download/24.17.29377.6/intel-opencl-icd_24.17.29377.6_amd64.deb
wget https://github.com/intel/compute-runtime/releases/download/24.17.29377.6/libigdgmm12_22.3.19_amd64.deb`

3. `sudo dpkg -i *.deb`

After that, Transcoding & HW/GPU Accerlation is working 🍀

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/2928#discussioncomment-9344576
